components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -347,11 +348,14 @@ define([
                  var cancerData = pedigree.GG.properties[i].cancers[cancer];
                  if (!cancerData.affected) {
                      output += "0\t";
+                     if (yob == "0") {
+                        warnMissingDOBUnaff = true;
+                     }
                  } else {
                      var ageAtDetection = cancerData.hasOwnProperty("numericAgeAtDiagnosis") && (cancerData.numericAgeAtDiagnosis > 0)
                                           ? cancerData.numericAgeAtDiagnosis : "AU";
                      output += ageAtDetection.toString() + "\t";
-                     if (yob == "0") {
+                     if (yob == "0" || ageAtDetection == "AU") {
```

Comment:
Hmm I see, is there a way I can do a test import to see what BOADICEA will be happy with?